### ISSUE TITLE
if someone still active here, please release the latest upstream package.

### ISSUE THREAD

**Issue by pinitik1906** [intel/intel-vaapi-driver#570]
anyone else and I, with an old computer, still have an issue about wl-drm inside of Wayland. video acceleration is a big help for us!

CPU: Intel(R) Core(TM) i5-4300U

**Comment by andrewpthompson** [comment]
Yes, please please make one final release containing the latest commits. So many downstreams are currently broken due to not having https://github.com/intel/intel-vaapi-driver/commit/d30e01329344858f3c84d0ef9c2b68cbde37bb9a and a release would help save so much time!
